charts/kube-otel-stack/values.yaml

```
@@ -1242,3 +1242,25 @@ prometheus-node-exporter:
     ## If true, create PSPs for node-exporter
     ##
     pspEnabled: false
+
+opampBridge:
+  enabled: false
+  image: "ghcr.io/open-telemetry/opentelemetry-operator/operator-opamp-bridge:main"
+  endpoint: "wss://opamp.lightstep.com/v1/opamp"
+  headers:
+    "Authorization": "<lightstep api key>"
```

best way to indicate that auth headers are possible without leaving these weird stubs in here?

---

```suggestion
    "Authorization": "bearer ${LS_OPAMP_API_KEY}"
```
I think maybe like this. The `${VARIABLE}` syntax is used for Lightstep access token in here

---

oh yeah when i wrote this i definitely forgot _everyone_ will need to include an auth header, its not optional. duh